tests/sensu-client-windows.pp

```diff
@@ -0,0 +1,9 @@
+# Note: Check http://repositories.sensuapp.org/msi/ for the latest version.
+class { '::sensu':
+  version           => '0.29.0-11',
+  rabbitmq_password => 'correct-horse-battery-staple',
+  rabbitmq_host     => '192.168.56.10',
+  rabbitmq_vhost    => '/sensu',
+  subscriptions     => 'all',
+  client_address    => $facts['networking']['ip'],
```

Comment:
This must be non-deterministic behavior in the  networking fact.  Joy.  In my vagrant vm $facts['networking']['ip'] resolves to 192.168.56.15.  I assume this resolves to 10.0.2.15 for you?